graphics/src/ColladaLoader.cc

```diff
@@ -2034,8 +2036,14 @@ void ColladaLoaderPrivate::LoadPolylist(tinyxml2::XMLElement *_polylistXml,
     }
     else if (semantic == "TEXCOORD")
     {
-      this->LoadTexCoords(source, texcoords, texDupMap);
-      inputs[TEXCOORD].insert(ignition::math::parseInt(offset));
+      int offsetInt = ignition::math::parseInt(offset);
+      if (inputs[TEXCOORD].find(offsetInt) == inputs[TEXCOORD].end())
+      {
+        int set = ignition::math::parseInt(polylistInputXml->Attribute("set"));
```

Comment:
We haven't been checking for the `set` attribute before, is there a chance we break some existing user that doesn't have `set`? Note that `parseInt` returns `NAN_I` if the string is empty.

Reply:
added check in 50c0e2e